networkx/algorithms/isomorphism/isomorphvf2.py

```diff
@@ -21,8 +21,8 @@
 as the logical AND of the two functions.
 
 To include a semantic check, the (Di)GraphMatcher class should be
-subclassed, and the semantic_feasibility() function should be
-redefined.  By default, the semantic feasibility function always
+subclassed, and the :func:`~networkx.isomorphism.isomorphvf2.GraphMatcher.semantic_feasibility`
```

Comment:
These are still not rendering as links in the docs, maybe they need to be `:meth:` roles?

Reply:
Urgghh, it still doesn't do the proper linking. Any other ideas ?